config/default.yml

```diff
@@ -954,6 +954,13 @@ RSpec/Rails/HaveHttpStatus:
   VersionAdded: '2.12'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/HaveHttpStatus
 
+RSpec/Rails/InferredSpecType:
+  Description: Identifies redundant spec type.
+  Enabled: pending
+  Safe: false
+  VersionAdded: '2.13'
```

Comment:
`<<next>>`, since it would be 2.14 already

Reply:
That's right. It's been a bit long time since I first made this pull request.
`<<next>>` is only mentioned at rubocop gem, so I thought that was the only convention there, but can it also be used rubocop-rspec gem?

Reply:
We're mimicking this convention, since not uncommon that PRs miss their initially targeted releases.
However, we don't formally describe this practice. I couldn't find this mentioned in rubocop, can you please help me find it, apart from the new cop generator spec? I'll be happy to document this properly.